.storybook/main.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+
+const baseConfig = require('./../.config/webpack/base.js')({
+    ts: {
+        transpileOnly: true
+    },
+    preprocessor: {
+        variables: {
+            mode: 'eager'
+        }
+    }
+});
+
+module.exports = {
+    stories: ['./../tests/visual/percy-storybook/**/*.percy.tsx'],
+    webpackFinal: async (config, { configType }) => {
+        // jerry rig everything
+        config.resolve.alias.core = path.resolve(__dirname, './../src/core'),
+        config.resolve.alias['dash-table'] = path.resolve(__dirname, './../src/dash-table')
+
+        config.module = baseConfig.module;
+
+        return config;
+    }
+};
```

Comment:
Storybook 6 brings totally new configuration files and opinions. Making it work with our own needs..

Reply:
@Marc-Andre-Rivet For any package using Storybook >=6, I imagine we now have to add

```javascript
config.resolve.alias['dash-table'] = path.resolve(__dirname, './../src/dash-table')
```

but specific to the path/package name?

Reply:
Yes. But the table is the only project using storybook.